src/Bundle/Resources/config/services/state.xml

```diff
@@ -95,15 +95,17 @@
             <tag name="sylius.state_provider" />
         </service>
 
-        <service id="Sylius\Component\Resource\Doctrine\Common\State\PersistProcessor">
+        <service id="Sylius\Resource\Doctrine\Common\State\PersistProcessor">
             <argument type="service" id="doctrine" />
             <tag name="sylius.state_processor" />
         </service>
+        <service id="Sylius\Component\Resource\Doctrine\Common\State\PersistProcessor" alias="Sylius\Resource\Doctrine\Common\State\PersistProcessor" />
```

Comment:
Is deprecation notice needed over here?

https://symfony.com/doc/current/service_container/alias_private.html#deprecating-service-aliases

Reply:
hum, this is not needed here, I have to remove that alias cause this is from 1.11. I forgot to remove that.

Reply:
thx for the doc, cause I will need that for future PRs soon.